DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -147,6 +171,92 @@ void L1TStage2uGT::bookHistograms(DQMStore::IBooker &ibooker, edm::Run const&, e
    prescaleFactorSet_ = ibooker.book2D("prescaleFactorSet", "uGT: Index of Prescale Factor Set vs. LS", numLS, 0., numLS_d, 25, 0., 25.);
    prescaleFactorSet_->setAxisTitle("Luminosity Segment", 1);
    prescaleFactorSet_->setAxisTitle("Prescale Factor Set Index", 2);
+
+  // Prescaled and Unprescaled Algo Trigger Bits
+  prescaled_first_collision_run_ = ibooker.book2D("prescaled_first_collision_run_", "uGT: Prescaled Algorithm Trigger Bits  vs. BX Number In Event", 5, -2.5, 2.5, prescaledAlgoBitName_.size(), -0.5, preAlgs_d-0.5);
+  prescaled_first_collision_run_->setAxisTitle("Bunch Crossing Number In Event", 1);
+  prescaled_first_collision_run_->setAxisTitle("Algorithm Trigger Bits + Names ", 2);
+  for(unsigned int algo=0; algo<prescaledAlgoBitName_.size(); algo++) {
+    prescaled_first_collision_run_->setBinLabel(algo+1,prescaledAlgoBitName_.at(algo).first+"("+std::to_string(prescaledAlgoBitName_.at(algo).second)+")",2);
+  }
+
+  den_prescaled_first_collision_run_ = ibooker.book2D("den_prescaled_first_collision_run_", "uGT: Prescaled Algorithm Trigger Bits Deno vs. BX Number In Event", 5, -2.5, 2.5, numAlgs, -0.5, numAlgs_d-0.5);
```

Comment:
The y axis range is wrong. It should only contain `prescaledAlgoBitName_.size()` bins like the numerator histogram.
Rename histogram to "prescaled_algo_first_collision_in_train_den".